ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Highlightings/FSharpErrorUtil.fs

```diff
@@ -47,3 +47,10 @@ let rec getUnusedExpr (expr: ISynExpr) =
         if isNotNull lastExpr then getUnusedExpr lastExpr else expr
 
     | _ -> expr
+
+let getAttributeSuffixRange (attribute: IAttribute) =
+    let attributeName = attribute.ReferenceName.ShortName
+    if not (attributeName.EndsWith("Attribute")) then DocumentRange.InvalidRange else
+
+    let shortAttributeName = attributeName.TrimFromEnd("Attribute")
+    attribute.ReferenceName.GetHighlightingRange().TrimLeft(shortAttributeName.Length)
```

Comment:
Extending the range from the end is probably easier to reason about, also a `null` check:
```suggestion
    let referenceName = attribute.ReferenceName
    if isNull referenceName || not (referenceName.ShortName.EndsWith("Attribute")) then DocumentRange.InvalidRange else

    referenceName.GetDocumentEndOffset().ExtendLeft("Attribute".Length)
```